job_spec/ARIA_RAIDER.yml

```diff
@@ -1,20 +1,14 @@
 ARIA_RAIDER:
   required_parameters:
-    - granules
+    - job_id
   parameters:
-    granules:
+    job_id:
       default:  '""'
```

Comment:
OH, is it because `granules` are always an array/list:
https://github.com/ASFHyP3/hyp3/blob/develop/job_spec/RTC_GAMMA.yml#L5-L10

But for the job definition, we want it to be a string, so that it's sent as a space-seperated string to the docker run command?
https://github.com/ASFHyP3/hyp3/blob/develop/job_spec/RTC_GAMMA.yml#L138

Which we do in start_execution_worker:
https://github.com/ASFHyP3/hyp3/blob/develop/apps/start-execution-worker/src/start_execution_worker.py

But I'm still not seeing why having `""` for the batch job definition is desirable over `None`...

@asjohnston-asf do you know why we provide a default of `'""'` for the granules parameter?

Reply:
Looking back through the git history, I think this is just something we carried along from way back in the day and isn't required. RTC_GAMMA, before INSAR_GAMMA was added, used to just be a string (not an array) and we provided the default `'""'`. Then, every parameter was required to have a default. That is, this:
```
{{ k }}: {{ v.get('default') or v['api_schema'].get('default') }}
```
was just:
```
{{ k }}: {{ v.['default'] }}
```

So when we switched to granules being a list, we made it so the default wasn't required anymore (would end up being `None` if not provided) but kept it for granules.

Reply:
I suggest we remove the default from `job_id` here. 

And maybe consider removing it from everything that we don't explicitly require it, which is only for optional parameters (just `bucket_prefix` right now). Also, it would be worth adding a comment on why the default is what it is when we leave it.